sdk/compiler/damlc/lib/DA/Cli/Damlc/Command/MultiIde/Handlers.hs

```diff
@@ -254,10 +262,27 @@ clientMessageHandler miState unblock bs = do
               LSP.FcDeleted -> do
                 shutdownIdeByHome miState home
                 handleRemovedPackageOpenFiles miState home
+                atomically $ onSubIde_ miState home $ \ideData -> ideData {ideDataIsFullDamlYaml = False}
               LSP.FcCreated -> do
-                handleCreatedPackageOpenFiles miState home
-                rebootIdeByHome miState home    
-              LSP.FcChanged -> rebootIdeByHome miState home
+                isFullDamlYaml <- shouldHandleDamlYamlChange <$> T.readFile (unPackageHome home </> "daml.yaml")
+                if isFullDamlYaml
+                  then do
+                    handleCreatedPackageOpenFiles miState home
+                    rebootIdeByHome miState home
+                  else shutdownIdeByHome miState home
```

Comment:
If a daml.yaml containing only an sdk version is created, we shutdown its IDE and don't update any state

Reply:
This would benefit from being added as a comment to the code. The PR comment below too.